README.md

```diff
@@ -282,7 +282,7 @@ Log in. If using the included terraform, this requires you SSH to the machine wi
 
 # Examples
 
-The examples are run through Jenkins making use of the Pipelines defined in [examples](examples). 
+The examples below are all executed through use of Jenkins Pipelines. The pipeline code can be found in [examples](examples). 
```

Comment:
This change is just rephrasing but the examples section in general might benefit from a statement about the practical use case statements for each example, such as:

## Example 1: Infrastructure Pipeline

_"I want to be able to rebuild my base operating system using code" _

---

## Example 2: Application Automation

_"I want to be able to deploy updates to my applications using code" _

Reply:
Resolved in #2